### PR TITLE
search certificates in parallel

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: searcher.go [hunk 1]
```diff
@@ -71,15 +72,31 @@ func (s *Searcher) SearchCluster(clusterID string) (Cluster, error) {
...
+
+			m.Lock()
+			defer m.Unlock()
+			err = fillTLSFromSecret(c.TLS, secret, clusterID, c.Cert)
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			return nil
+		})
```
Comment (tuommaki) [inline searcher.go]: Is `g.Wait()` missing in these blocks?

Comment (xh3b4sd) [reply searcher.go]: Yes, added.

**Review by tuommaki — APPROVED** [review]
LGTM when tested to work

**Review by rossf7 — APPROVED** [review]
LGTM when tested.

**Comment by xh3b4sd** [comment]
Tested on `ginger`.

**Pull request merged**